apps/scan/frontend/src/screens/election_manager_screen.test.tsx

```diff
@@ -370,7 +370,7 @@ test('switching to test mode when ballots have been counted', async () => {
   ).toEqual('false');
 });
 
-test('machine cannot be switched to test mode if CVR sync is required and ballots have been counted', async () => {
+test('machine cannot be switched to test mode if CVR sync is required', async () => {
```

Comment:
Updating tests to make it clear that CVR sync can't be required if no ballots have been counted